.mergify.yml

```diff
@@ -1,4 +1,17 @@
 pull_request_rules:
+  # if there is a conflict in a backport PR, ping the author to send a proper backport PR
+  - name: ping author on conflicts
+    conditions:
+      - conflict
+    actions:
+      comment:
+        message: This pull request is now in conflict, please @{{author}} rebase it.
+  # remove stale reviews
+  - name: remove outdated reviews
```

Comment:
What are the rules for removing stale reviews? Is there some timeout if the one requesting changes doesn't respond to the author?

Reply:
The idea is to remove the reviews when new code is pushed. Mergify doc says:

> When a pull request is updated, GitHub does not remove the (possibly) outdated reviews approvals or changes request. It’s a good idea to remove them as soon as the pull request gets updated with new commits.

I think it's nice to reset everything after each push so that we are not stuck. We have seen the case many times where we wait for someone to review again but the changes have been addressed but we can't merge because the outdated request change.

Makes sense?

Reply:
If the changes requested flag is removed every time someone pushes, that seems too aggressive. The one requesting changes may have follow up questions. If someone else approves the PR before they even have a chance to review, the PR may merge too soon. I'd prefer to keep the current ability where maintainers can dismiss reviews if needed.

Reply:
Fair enough.